DataRepo/formats/dataformat.py

```diff
@@ -768,6 +768,41 @@ def getModelInstance(self, mdl):
                 )
         return mdl
 
+    def getModelFromInstance(self, mdl_instance):
```

Comment:
NB: Type hints might make it more obvious that `mdl_instance` is actually the name of a model instance and not an actual instance.
```suggestion
    def getModelFromInstance(self, mdl_instance: str):
```

Reply:
Since I'm going to attempt to rebase and redo the work in this PR, I'll let this and the other issues sit unresolved until the rebase strategy is complete.  If that goes off without a hitch, I'll close this PR.

Reply:
There is no need to redo any work. Just leave this PR alone for now and once we have the base set to something with the "extra" commits, they will go away from the PR.

Reply:
Are you saying that this wacky commit history will get cleaned up if I rebase this branch?  Or are you saying that the diff will look better only?

Reply:
OK, so everything has been merged into the feature branch, but my inferences of a few of your comments feel conflicting.  Do you want me to rebase this branch or the feature branch that this merges into?  I feel like it has to be this branch, because rebasing will cause the same problems I fixed in this branch.  But you said "once we have the base set to something with the "extra" commits", which implies that you want me to rebase the feature branch, but you also said I don't need to do any extra work.  Does that mean you want me to leave that branch broken and then merge this into that to fix it?

Reply:
...rebasing that feature branch will involve resolving conflicts, which is work I did already in this branch.  So I'm pretty confused as to what path forward it is you're suggesting.